### PR TITLE
Tweak zero

### PR DESCRIPTION
**Pull request by colinrford**
Here are some minor tweaks I've made since **Sprint 1** in hopes to unify our app a little bit. This is _not_ complete, so please do not merge just yet! This is the time for you all to look at this branch and critique. I would like to have this PR merged by **Tuesday evening**, if possible.

Initially, I was going to change everything from activities over to View Models. We changed our mind on that. Instead, I am working on converting activities to fragments. 

I have also been working on adding the navigation drawer to everyone's module. @brandonschabell and @MrJohnson, we're going to need to work together a little bit on this so that I don't wreck your branches from merging this PR. @avaidyam, @Wall-Dough and @Brahmancer, I do not believe this branch will affect much of what you all are doing, but I could be wrong so please give me a heads up if so. 

This branch also will disable tablet use and disable landscape use in our app, indefinitely. 

In short, _these are small yet crucial tweaks that will allow development for our app to proceed smoothly!_

### CODE REVIEW

File: Health/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -56,36 +75,46 @@
...
         <!-- For Steps testing. -->
         <activity
             android:name=".steps.StepsViewModel"
             android:label="@string/steps_module"
-            android:theme="@style/AppTheme.NoActionBar"/>
+            android:theme="@style/AppTheme.NoActionBar"
+            android:screenOrientation="portrait" />
+
+        <!-- For Workout testing. -->
+        <activity
```
Comment (avaidyam) [inline Health/src/main/AndroidManifest.xml]: These need to be killed.

File: Health/src/main/java/com/sciencesquad/health/ClockActivity.java [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
-import com.sciencesquad.health.prescriptions.PrescriptionModel;
+import com.sciencesquad.health.steps.StepsViewModel;
 import com.sciencesquad.health.ui.EmergencyNotification;
+import com.sciencesquad.health.workout.WorkoutActivity;
+import com.sciencesquad.health.activity.MapsActivity;
 
-import java.util.Calendar;
-
-public class ClockActivity extends BaseActivity {
+public class ClockActivity extends BaseActivity implements
```
Comment (avaidyam) [inline Health/src/main/java/com/sciencesquad/health/ClockActivity.java]: `Fragment`?

File: Health/src/main/java/com/sciencesquad/health/activity/MapsActivity.java [hunk 3]
```diff
@@ -314,4 +323,43 @@ public void onLocationChanged(Location location) {
         handleNewLocation(location);
 
     }
+
+    /**
+     * I have not been tested :-)
+     */
+    @SuppressWarnings("StatementWithEmptyBody")
+    @Override
+    public boolean onNavigationItemSelected(MenuItem item) {
```
Comment (avaidyam) [inline Health/src/main/java/com/sciencesquad/health/activity/MapsActivity.java:332]: Please don't do this. :cry: Remove all of these from everything except `MainActivity`.

File: Health/src/main/java/com/sciencesquad/health/ClockActivity.java (hunk 2 above)
Comment (colinrford) [inline Health/src/main/java/com/sciencesquad/health/ClockActivity.java]: @avaidyam not yet. this was written pre-fragment decision
